tests/test_host.py

```diff
@@ -7,3 +7,9 @@
 def test_should_use_domain_in_endpoint_url(domain):
     host.DOMAIN = domain
     assert host.endpoint("/path") == f"https://api.up42.{domain}/path"
+
+
+@pytest.mark.parametrize("domain", ["xyz", "abc"])
+def test_should_use_domain_in_token_endpoint_url(domain):
```

Comment:
Why would we test the domain only? If anything it would make more sense to me to test that the entire URL. And in the end we're testing it's right by using the fixture `fixture_globals.TOKEN_ENDPOINT`.